src/pages/BudgetApp/Budget.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { PageHeader, BudgetContainer, StyledBudgetContainer } from "../../styling/Styled.jsx";
+import BudgetTotal from './components/BudgetTotal';
+import RemainingBudget from './components/RemainingTotal';
+import ExpensesTotal from './components/ExpensesTotal';
+import ExpenseList from './components/ExpenseList.jsx';
```

Comment:
Imports shouldn't need the .jsx at the end, please delete it from this line and for:
'./components/AddExpenseForm.jsx';
'./components/charts/GaugeChart.jsx';
'./components/charts/BarChart.jsx';